packages/axelar-wasm-std/src/voting.rs

```diff
@@ -220,15 +226,26 @@ impl Poll for WeightedPoll {
             return Err(Error::PollNotEnded);
         }
 
-        if self.status == PollStatus::Finished {
-            return Err(Error::PollNotInProgress);
-        }
+        let consensus_participants = self
+            .participation
+            .iter()
+            .filter_map(|(address, participation)| {
+                participation.vote.as_ref().and_then(|vote| {
+                    if *vote == results {
+                        Some(address.to_owned())
+                    } else {
+                        None
+                    }
+                })
+            })
+            .collect();
```

Comment:
I don't know, I like the idea that `PollResult` returns everything needed by the `execute` module to know how to react, I don't see much of an issue since the computation its coupled with the votes result. Computing it a the callsite will need access to some of the fields in `WeightedPoll` instead of accessing the trait `Poll`, might want to add a new function in the trait and compute there instead, but for simplicity I would like to keep it as it is unless someone has strong opinion about an alternative